### PR TITLE
Exposed `status` and `block` RPC

### PR DESCRIPTION
**Pull request by frol**

### CODE REVIEW

**Comment by frol** [comment]
@ilblackdragon @vgrichina please, review.

**Comment by frol** [comment]
1. Sure
2. Yes, the snake case style is from the Nearcore side, so it cannot be changed without manual renaming.

**Comment by frol** [comment]
@vgrichina I have added the tests and rebased (there has been a merge conflict). Please, review.

File: test/providers.test.js [hunk 1]
```diff
@@ -1,6 +1,20 @@
 
 const nearlib = require('../lib/index');
 
+test('json rpc fetch node status', async () => {
+    const config = Object.assign(require('./config')(process.env.NODE_ENV || 'test'));
```
Comment (vgrichina) [inline test/providers.test.js]: looks like there is common setup (`config`, `provider`) that you want to move to `beforeEach`

**Pull request merged**